### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by alexkart**

### CODE REVIEW

**Comment by peter279k** [comment]
The problem is as follows:

```
This package requires php ^7.1 but your PHP version (8.0.0-dev) does not satisfy that requirement.
```

The reason is about the composer PHP version checking, and the `8.0.0-dev` is unstable version.

To fix this issue temporarily, it should use the ` --ignore-platform-reqs` when using the `composer` command.

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ php:
   - 7.1
   - 7.2
   - 7.3
+  - 8.0snapshot
```
Comment (peter279k) [inline .travis.yml:8]: It should be  good to add the `php-7.4` version test during Travis CI build.

**Comment by peter279k** [comment]
The PHPUnit tests are done, but the static code analyse for `Phan` has been problematic.

To fix that issue temporarily, I think it should let `php-8.0dev` run without `Phan` checking.

**Comment by peter279k** [comment]
BTW, after checking the [Phan .travis.yml](https://github.com/phan/phan/blob/master/.travis.yml) file, it doesn't do `php-8.0.0-dev` test currently during Travis CI building.

**Comment by alexkart** [comment]
@peter279k thanks for the help.

**Pull request merged**